src/indexation/pht.cpp

```diff
@@ -27,37 +27,27 @@ namespace indexation {
 /**
  * Output the blob into string and readable way
  *
- * @param bl   : Blob to print
+ * @param bl   A Blob
+ * @param nbr  The number of bits to consider.
  *
- * @return string that represent the blob into a readable way
+ * @return string that represent the blob
  */
-static std::string blobToString(const Blob &bl) {
+static std::string blobToString(const Blob &bl, size_t nbr = 0) {
```

Comment:
> method name is not accurate. It doesn't convert a Blob to a string.

May be `firstBitsToString`?

> nbr should not be optional (we manipulate bits, not bytes), allowing to simplify line 37. One could argue nbr is not a good argument name (type size_t already tells it's a number).

I agree.

> optional, at your preference: might be private, non-static in Prefix, using size_ directly (use and design are specific to Prefix).

If you look at the historic of the files `pht.cpp` and `pht.h`, you'll see that it was first the case (see 2d8fe502f29aaddced8584283f2db370e9ee1a85). Not much is said as to why it was changed. I guess that we found that the function was not depending on the state of the instance of Prefix and could apply to other objects directly like `Blob`. However, I think that we can limit ourselves to the `Prefix` class for our needs as of now, like you suggest.